apps/common-app/src/examples/RuntimeTests/tests/plugin/recursion.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect } from 'react';
 import { View } from 'react-native';
-import { useSharedValue, runOnUI, runOnJS } from 'react-native-reanimated';
+import { useSharedValue, runOnUI } from 'react-native-reanimated';
 import { render, wait, describe, getRegisteredValue, registerValue, test, expect } from '../../ReJest/RuntimeTestsApi';
```

Comment:
This is very cool. Could type annotations be added to the test that test file level "worklet" annotations? To be on the safe side?